ledger/sandbox/BUILD.bazel

```diff
@@ -20,6 +20,7 @@ load(
 
 compileDependencies = [
     # this has to come first other as somehow a different instance of grpc-core.jar
```

Comment:
please move the comment to the right place

Reply:
Done